src/Illuminate/Support/Number.php

```diff
@@ -141,30 +141,30 @@ public static function fileSize(int|float $bytes, int $precision = 0, ?int $maxP
      * @param  int|null  $maxPrecision
      * @return string
      */
-    public static function forHumans(int|float $number, int $precision = 0, ?int $maxPrecision = null)
+    public static function forHumans(int|float $number, int $precision = 0, ?int $maxPrecision = null, ?string $locale = null)
```

Comment:
Since this feature already released and class not final — this is a breaking change.

Reply:
I'm not sure it is. Generally adding parameters to a method signature is not considered breaking in my personal experience with SemVer. Furthermore, while only tangentially related, Laravel does not include named arguments in their [backwards compatibility guidelines](https://laravel.com/docs/10.x/releases#named-arguments).